aws-junit/src/main/java/zipkin/junit/aws/AmazonSQSRule.java

```diff
@@ -136,7 +137,6 @@ private void sendSpansInternal(List<Span> spans) {
 
   private static List<Span> fromBase64(String base64) {
     byte[] bytes = Base64.decode(base64);
-    return Codec.THRIFT.readSpans(bytes);
+    return SpanDecoder.DETECTING_DECODER.readSpans(bytes);
```

Comment:
this works with all the formats and prevents us from needing to access internal classes (for the not yet released span2 type)